website/versioned_docs/version-0.63/hermes.md

```diff
@@ -70,9 +70,11 @@ $ npx react-native run-android --variant release
 
 This will compile JavaScript to bytecode during build time which will improve your app's startup speed on device.
 
-## Debugging Hermes using Google Chrome's DevTools
+## Debugging JS on Hermes using Google Chrome's DevTools
 
-Hermes supports the Chrome debugger by implementing the Chrome inspector protocol. This means Chrome's tools can be used to directly debug JavaScript running on Hermes, on an emulator or device.
+Hermes supports the Chrome debugger by implementing the Chrome inspector protocol. This means Chrome's tools can be used to directly debug JavaScript running on Hermes, on an emulator or on a real, physical, device.
+
+> Note that this is very different with the "Remote JS Debugging" from the In-App Developer Menu documented in the [Debugging](https://reactnative.dev/docs/debugging#debugging-using-a-custom-javascript-debugger) section, which actually runs the JS code on Chrome's V8 on your development machine (laptop or desktop).
```

Comment:
Please use the internal link format (`(debugging#debugging-using-a-custom-javascript-debugger)`) otherwise link will be always leading to the docs page set as `current`, regardless of the version in which user browse the website.